### PR TITLE
vistafonts-cht: init

### PR DESCRIPTION
**Pull request by AtkinsChang**
###### Motivation for this change
`vistafonts-cht`, font from vista for CJK characters.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/data/fonts/vista-fonts-cht/default.nix [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    # Set up no-op font configs to override any aliases set up by
+    # other packages.
+    mkdir -p $out/etc/fonts/conf.d
+    substitute ${./no-op.conf} $out/etc/fonts/conf.d/30-msjhenghei.conf \
+      --subst-var-by fontname "Microsoft JhengHei"
+  '';
+
+  sha256 = "00v5h8gjm6pby8lbvm03sn29302am91vx93gyv8i72id63mbmy2j";
+
+  meta = {
```
Comment (alexarice) [inline pkgs/data/fonts/vista-fonts-cht/default.nix]: It is usual to use `with lib;` here to avoid having to specify lib everywhere in the meta

File: maintainers/maintainer-list.nix [hunk 2]
```diff
@@ -522,6 +522,11 @@
     github = "aszlig";
     name = "aszlig";
   };
+  atkinschang = {
```
Comment (alexarice) [inline maintainers/maintainer-list.nix]: Adding a maintainer should be done in a separate commit

**Comment by AtkinsChang** [comment]
@alexarice rebased into two commits

**Comment by AtkinsChang** [comment]
@edolstra would you help with this years old PR?

**Comment by AtkinsChang** [comment]
@SuperSandro2000 would you help with this years old PR?

File: pkgs/data/fonts/vista-fonts-cht/default.nix [hunk 3]
```diff
@@ -0,0 +1,42 @@
+{ lib, stdenvNoCC, fetchurl, cabextract }:
+
+
+stdenvNoCC.mkDerivation {
```
Comment (SuperSandro2000) [inline pkgs/data/fonts/vista-fonts-cht/default.nix:4]:
```suggestion
{ lib, stdenvNoCC, fetchurl, cabextract }:

stdenvNoCC.mkDerivation {
```
